### PR TITLE
pnpm 

### PR DESCRIPTION
**Pull request by ilyigna**
bad alien using npm is bad very bad

### CODE REVIEW

**Comment by ilyigna** [comment]
how do i make this draft cuh

**Review by splatterxl — CHANGES_REQUESTED** [review]
Please add a pnpm lockfile and amend the CI actions to use pnpm

**Comment by dolfies** [comment]
Done

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -12,13 +12,13 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: 16
-          cache: npm
+          cache: pnpm
```
Comment (splatterxl) [inline .github/workflows/test.yaml:16]: Maybe use the pnpm workflow here too?

**Comment by ilyigna** [comment]
i'll fucking find where eslint developers live.

**Review by dolfies — APPROVED** [review]

**Comment by ilyigna** [comment]
openssl is messed up in v18, that's why theres a error

**Comment by dolfies** [comment]
It was broken before this PR so out of scope

**Pull request merged**